File: ClientApp/models/persistency/retroAction.ts
import { CardBase } from "./cardBase";
import { User } from "./user";

export enum RetroActionStatus {
    New = 0,
    Open = 1,
    Done = 3
}

export class RetroAction {
    public id: number = 0;
    public description: string = "";
    public inChargeTo: User|null = null;
    public whoChecks: User|null = null;
    public card: CardBase;
    public status: RetroActionStatus = RetroActionStatus.New;

    public constructor (card : CardBase){
        this.card = card;
    }
}

File: BusinessLogic/Models/Persistency/RetroAction.cs
namespace kRetro.BusinessLogic.Models.Persistency
{
    public enum RetroActionStatus{
        New=0,
        Open=1,
        Done=3
    }
    public class RetroAction
    {
        public int Id {get;set;}
        public string Description {get;set;}
        public User InChargeTo {get;set;}
        public User WhoChecks {get;set;}
        public CardBase Card {get;set;}
        public RetroActionStatus Status {get;set;}
    }
}